### ISSUE TITLE
Neural Network architecture when using visual observation 

### ISSUE THREAD

**Issue by fluffymudkip** [Unity-Technologies/ml-agents#811]
I was wondering if anybody knows how the neural network architecture will look like if i implement a camera and use various vector obeservations. 
Am i correct in assuming that with the hyperparameters below the visual observations is feed into a width*heigth input CNN which then is feed into a normal feed-forward NN, where the inputs for the FFNN will be the outputs of the CNN+the vector observations? 

    learning_rate: 5.0e-5
    batch_size: 64
    normalize: false
    num_layers: 2
    hidden_units: 128
    beta: 5.0e-3
    normalize: false
    gamma: 0.95
    buffer_size: 256
    max_steps: 5.0e9
    summary_freq: 1000
    time_horizon: 64
    use_recurrent: false

**Comment by MarcoMeter** [comment]
You can checkout the method [create_visual_encoder()](https://github.com/Unity-Technologies/ml-agents/blob/master/python/unitytrainers/models.py#L83)

There are two convolutional layers setup originally.

**Comment by xiaomaogy** [comment]
Thank you for the discussion. We are closing this issue due to inactivity. Feel free to reopen it if you’d like to continue to discussion though.
